roles/preflight-checks/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+skip_prechecks_all: false
```

Comment:
why can;t we use tags to skip-tags rather than define variables?

I don;t think we should be selective here. Just add tags to site,yml for this play and one can use --skip-tags if they know what they doing.